telldus-core/common/EventHandler_unix.cpp

```diff
@@ -79,7 +79,7 @@ bool EventHandler::waitForAny() {
 	while (!d->isSignalled) {
 		timeval now;
 		gettimeofday(&now, NULL);
-		uint64_t abstime_ns_large = now.tv_usec*1000 + 60000000000;  // add 60 seconds wait (5 seconds before)?
+		suseconds_t abstime_ns_large = now.tv_usec*1000 + 60000000000;  // add 60 seconds wait (5 seconds before)?
```

Comment:
The calculation will overflow 32 bits so we need it to be 64 bits, even on 32-bit platforms.